### PR TITLE
Enhancement display functions

### PR DESCRIPTION
**Pull request by DavidMansolino**
Linked with omichel/webots#4051

### CODE REVIEW

**Comment by DavidMansolino** [comment]
:+1:

File: reference/java-api.md [hunk 1]
```diff
@@ -151,7 +153,7 @@ The following tables describe the Java classes and their methods.
 | &nbsp;&nbsp; public void [fillOval](display.md#wb_display_draw_pixel)(int cx, int cy, int a, int b);                                                   |
 | &nbsp;&nbsp; public void [fillPolygon](display.md#wb_display_draw_pixel)(int[] x, int[] y);                                                            |
 | &nbsp;&nbsp; public [ImageRef](#java_image_ref) [imageCopy](display.md#wb_display_image_new)(int x, int y, int width, int height);                     |
-| &nbsp;&nbsp; public void [imagePaste](display.md#wb_display_image_new)([ImageRef](#java_image_ref) ir, int x, int y);                                  |
+| &nbsp;&nbsp; public void [imagePaste](display.md#wb_display_image_new)([ImageRef](#java_image_ref) ir, int x, int y, boolean blend);                   |
```
Comment (omichel) [inline reference/java-api.md]: Default value is missing.

Comment (DavidMansolino) [inline reference/java-api.md]: I don't think Java support default argument.

**Comment by DavidMansolino** [comment]
:+1:

**Comment by omichel** [comment]
:+1:

**Pull request merged**